osidb/tests/test_cvss.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+from django.core.exceptions import ValidationError
+
+from osidb.models import CVSS
+from osidb.tests.factories import (
+    AffectCVSSFactory,
+    AffectFactory,
+    FlawCVSSFactory,
+    FlawFactory,
+)
+
+pytestmark = pytest.mark.unit
+
+
+class TestCVSS:
+    def test_create(self):
+        flaw = FlawFactory()
+        affect = AffectFactory(flaw=flaw)
+
+        flaw_cvss2 = FlawCVSSFactory(
+            flaw=flaw, version=CVSS.CVSSVersion.VERSION2, issuer=CVSS.CVSSIssuer.REDHAT
+        )
+        flaw_cvss3 = FlawCVSSFactory(flaw=flaw, version=CVSS.CVSSVersion.VERSION3)
+
+        affect_cvss2 = AffectCVSSFactory(
+            affect=affect, version=CVSS.CVSSVersion.VERSION2
+        )
+        affect_cvss3 = AffectCVSSFactory(
+            affect=affect,
+            version=CVSS.CVSSVersion.VERSION3,
+            issuer=CVSS.CVSSIssuer.NIST,
+        )
+
+        all_flaw_cvss_scores = flaw.cvss_scores.all()
+        assert len(all_flaw_cvss_scores) == 2
+        assert flaw_cvss2 in all_flaw_cvss_scores
+        assert flaw_cvss3 in all_flaw_cvss_scores
+
+        all_affect_cvss_scores = affect.cvss_scores.all()
+        assert len(all_affect_cvss_scores) == 2
+        assert affect_cvss2 in all_affect_cvss_scores
+        assert affect_cvss3 in all_affect_cvss_scores
+
+        # Test flaw/affect-version-issuer uniqueness constraint.
+        with pytest.raises(ValidationError) as e:
```

Comment:
nitpick: we try to standardize on `match` parameter instead of the later `assert`

Reply:
...it is on multiple places here

Reply:
Changed